### DISCUSSION TITLE
Modeling water flow through a channel by thermal hydraulics module

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/moose#20626] [Q&A Modules: General]
Can I use THM to model water flow through a channel?  I tried to just replace the IdealGasFluidProperties with Water97FluidProperties, but an error occured saying that  s_from_v_e has not been implemented.

**Comment by GiudGiud** [comment]
@andrsd

**Comment by andrsd** [comment]
It is possible, but not with `Water97FluidProperties` class. This class is missing implementation that is needed for the compressible Navier-Stokes formulation w/rDG we are using inside THM.
Out of the box you should be able to use `IdealGasFluidProperties` and `StiffenedGasFluidProperties`.
